NitroxPatcher/Patches/Vehicle_Awake_Patch.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Reflection;
+using Harmony;
+using NitroxClient.GameLogic;
+using NitroxClient.GameLogic.Helper;
+using NitroxModel.Core;
+
+namespace NitroxPatcher.Patches
+{
+    public class Vehicle_Awake_Patch : NitroxPatch
+    {
+        public static readonly Type TARGET_CLASS = typeof(Vehicle);
+        public static readonly MethodInfo TARGET_METHOD = TARGET_CLASS.GetMethod("Awake", BindingFlags.Public | BindingFlags.Instance);
+
+        public static void Postfix(Vehicle __instance)
+        {
+            NitroxServiceLocator.LocateService<Vehicles>().Add(__instance);
```

Comment:
Is this for the console command?  I believe we are already syncing when the vehicle constructor creates a vehicle.  That code path might execute at the same time as this and cause a conflict. We may have to delete ConstructorInput_OnCraftingBegin_Patch.

Reply:
for the moment (Mobile Vehicle Bay) no sync the  ConstructorInput_OnCraftingBegin_Patch no work because a missing owner went repair that can delete Vehicle_Awake_Patch